packages/react-components/src/components/MessageThread.tsx

```diff
@@ -163,6 +163,12 @@ export interface MessageThreadStrings {
   participantJoined: string;
   /** String for participants left */
   participantLeft: string;
+  /** String for edited tag of a message */
```

Comment:
nit: Could improve the description here "Tag shown on a message that has been edited".